pkg/kubelet/dockershim/docker_service.go

```diff
@@ -140,18 +142,21 @@ func NewDockerService(client dockertools.DockerInterface, seccompProfileRoot str
 	glog.Infof("Docker cri networking managed by %v", plug.Name())
 
 	// NOTE: cgroup driver is only detectable in docker 1.11+
-	var cgroupDriver string
+	cgroupDriver := defaultCgroupDriver
 	dockerInfo, err := ds.client.Info()
 	if err != nil {
-		glog.Errorf("failed to execute Info() call to the Docker client: %v", err)
-		glog.Warningf("Using fallback default of cgroupfs as cgroup driver")
```

Comment:
@derekwaynecarr, the code doesn't actually set the driver to cgroupfs. I fix that in this commit.